### DISCUSSION TITLE
Welcome to aws-sdk-ruby Discussions!

### DISCUSSION THREAD

**Discussion by mullermp** [aws/aws-sdk-ruby#2569] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mullermp** [comment]
Hello, world!

**Reply by WebDeverDan** [reply]
Hey there! Do you have any examples of how to use Sigv4 and aws-sdk in an Httparty request? I'm having a hard time making sense of how to replicate postman's aws signature request. Thank you!!

**Reply by mullermp** [reply]
Hey. I'm not aware that enabling discussions feature made this self post... Could you please ask this in a new discussions topic so that we can mark accepted answers?

**Reply by WebDeverDan** [reply]
Absolutely. It's my first time using the discussions feature here, sorry!
